### DISCUSSION TITLE
table_mode other

### DISCUSSION THREAD

**Discussion by mvolkmann** [nushell/nushell#3170] [Q&A] [answered]
What does setting `table_mode` to `other` do?

**Comment by fdncred** [comment, marked as answer]
According to the code [here](https://github.com/nushell/nushell/blob/2c89a228d54ab70b25caecdeeb2c2d4daf145e6d/crates/nu-command/src/commands/table/options.rs#L49-L66) `other` is not valid so it defaults to `compact`.

**Reply by mvolkmann** [reply]
Okay. I guess the docs here are out of date, because it lists "other" as one of the options.
https://www.nushell.sh/book/configuration.html#nushell-configuration-with-config-toml
